src/ngeo.js

```diff
@@ -42,6 +39,9 @@ goog.require('ol.render.Feature');
 goog.require('ol.source.VectorTile');
 /** @suppress {extraRequire} */
 goog.require('ol.style.AtlasManager');
+// import 'angular-gettext/dist/angular-gettext';
+// import 'angular-ui-date/dist/date';
+// import 'angular-float-thead/angular-floatThead';
```

Comment:
Last two must be moved to the files that make use of them.

Reply:
Consider moving // import 'angular-gettext/dist/angular-gettext'; to the applications / examples.
=> Swisstopo uses a wrapper implementing the angular-gettext API but with a custom implementation.
It may be a common need.